### DISCUSSION TITLE
Is there any way to get the mouse position during dragging events?

### DISCUSSION THREAD

**Discussion by yf-hk** [clauderic/dnd-kit#222] [Q&A]
I'm working on a project which requires using mouse position to make changes to the scale level of the overlay (`active.rect` is not quite suitable for my use case).
I've done a bit of research but with no luck.
Is it currently doable? If not, can we add that in?
Thanks in advance

**Comment by clauderic** [comment]
Internally the library keeps track of `pointerCoordinates`, though it's not something that is exposed externally at the moment. Could you elaborate a bit more on your use-case? Do you really need the pointer coordinates?

**Reply by yf-hk** [reply]
Yes, I'm dragging a photo into a canvas. I need to know when excactly the mouse enters the frame.

**Reply by yf-hk** [reply]
Also, if mouse position can be passed in, problems in this issue can be solved easily #122
Currently there's no way to do it

**Reply by kokipsiho** [reply]
We are having the same issues described in #122. I'm working on pointer intersection detection which I believe would solve most use cases where DragOverlay is of different size. The approach is similar to the example in #127 but I'm struggling to get library events, not just native mouse move events. Ideally, using `getEventCoordinates` function to get normalized coordinates would cover any sensor type. Any way to get event object in custom collision detection?
